hail/python/hailtop/gear/auth/utils.py

```diff
@@ -0,0 +1,29 @@
+import secrets
+import base64
+
+
+async def create_user(dbpool, spec):
```

Comment:
So this doesn't actually create a user, it inserts one. This also seems to overlap with my [create user PR](https://github.com/hail-is/hail/pull/6619). How would you like to handle that?